src/lib/components/input/input.module.scss

```diff
@@ -69,3 +69,9 @@ input[type='date']:valid:before {
     color: themed('lavender');
   }
 }
+
+.input[readonly], .input[readonly]:focus {
+  background: $grey-30;
```

Comment:
We should use themefied values instead of the variable directly...because on website the background color is different from portal